### PR TITLE
chore(package): update eslint-config-steelbrain to version 4.0.1

### PR DESCRIPTION
**Pull request by Arcanemagus**
Closes #448

### CODE REVIEW

File: lib/commands.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* @flow */
 
 import invariant from 'assert'
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
```
Comment (steelbrain) [inline lib/commands.js]: Couldn't we add exceptions for these in the local eslint config? 😅

File: lib/panel/component.js [hunk 2]
```diff
@@ -7,18 +7,37 @@ import type Delegate from './delegate'
...
+        return row[column]
+    }
+  }
   constructor(props: Object, context: ?Object) {
     super(props, context)
     this.state = {
       messages: this.props.delegate.filteredMessages,
     }
   }
+  state: {
```
Comment (steelbrain) [inline lib/panel/component.js]: This part of the PR conflicts with flow update PR (which I think should be merged first)

Comment (Arcanemagus) [reply lib/panel/component.js]: It's actually not a conflict, this is a move due to an ESLint rule.

**Comment by Arcanemagus** [comment]
Actually these rules don't support the config that would be needed to do this correctly, I've looked into it before. You can either do individual ignore statements like this or turn off the rule entirely (which leads to the possibility of things they _would_ catch slipping through).

`import/no-unresolved` is the only one that can be [configured](https://github.com/AtomLinter/linter-eslint/blob/1427e52364351cbbe5ca433c23c6e8a23f72bfd3/package.json#L159).

**Comment by Arcanemagus** [comment]
@steelbrain Updated to configure the one rule that can be, let me know if you want to instead just disable the rules entirely.

**Pull request merged**